src/scripts/install.sh

```diff
@@ -0,0 +1,70 @@
+if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+
+if [ ! "$(which aws)" ] || [ "$PARAM_AWS_CLI_OVERRIDE" = 1 ]; then
+    # setup
+    if [ ! "$PARAM_AWS_CLI_VERSION" = "latest" ]; then export AWS_CLI_VER_STRING="-$PARAM_AWS_CLI_VERSION"; fi
+
+    # Uninstall existing AWS CLI if override is enabled.
+    if [ "$PARAM_AWS_CLI_OVERRIDE" = 1 ]; then
+        AWS_CLI_PATH=$(which aws)
+        if [ -n "$AWS_CLI_PATH" ]; then
+            EXISTING_AWS_VERSION=$(aws --version)
+            echo "Uninstalling ${EXISTING_AWS_VERSION}"
+            # shellcheck disable=SC2012
+            AWS_SYMLINK_PATH=$(ls -l "$AWS_CLI_PATH" | sed -e 's/.* -> //')
+            $SUDO rm -rf "$AWS_CLI_PATH" "$AWS_SYMLINK_PATH" "$HOME/.aws/" "/usr/local/bin/aws" "/usr/local/bin/aws_completer" "/usr/local/aws-cli"
+        else
+            echo "No AWS install found"
+        fi
+    fi
+
+    echo "Installing AWS CLI v2"
+    # Platform check
+    if uname -a | grep Darwin; then
+        export SYS_ENV_PLATFORM=darwin
+    elif uname -a | grep Linux; then
+        export SYS_ENV_PLATFORM=linux
+    elif uname -a | grep aarch64; then
```

Comment:
These if/else blocks are testing two different things. OS is separate from architecture. The first two clauses test OS while the 3rd test architecture. In this scenario, on CircleCI today, the third will never match because our ARM image is also Linux.

OS and architecture are a matrix.

Reply:
Excellent point, dummy mistake lol. It has been corrected, let me know what you think. Also, at the moment, the CLI will likely never install anyway on arm simply because it's pre-installed on that image. However, this orb does have the ability to uninstall and reinstall if you wish to change the version, so its possible.